File: Bakery.Infrastructure.Web/Buns/InputModels/GetBunsInputModel.cs
namespace Bakery.Infrastructure.Web.Buns.InputModels;

/// <summary>
/// Модель входных данных для получения контактов.
/// </summary>
public class GetBunsInputModel
{
    /// <summary>
    /// Лимит контактов.
    /// </summary>
    public int Limit { get; init; } = 100;

    /// <summary>
    /// Смещение.
    /// </summary>
    public int? Offset { get; init; }
}

File: Bakery.Client/src/app/services/models/input-models/get-buns-input-model.ts
/**
 * Интерфейс, представляющий входную модель для получения булочек.
 */
export interface GetBunsInputModel {
  /**
   * Лимит.
   */
  limit: number;

  /**
   * Смещение.
   */
  offset?: number;
}
